osgp/protocol-adapter-dlms/osgp-dlms/src/test/java/org/opensmartgridplatform/dlms/services/Protocol.java

```diff
@@ -12,12 +12,12 @@
 @Getter
 public enum Protocol {
   DSMR_2_2("DSMR", "2.2", 32, 2, 4, 6, 14, 0, 0, 0, 0, false, false),
-  DSMR_4_2_2("DSMR", "4.2.2", 58, 11, 27, 6, 14, 6, 13, 5, 6, true, false),
-  SMR_4_3("SMR", "4.3", 59, 11, 27, 6, 14, 6, 13, 6, 7, true, false),
-  SMR_5_0_0("SMR", "5.0.0", 62, 11, 27, 6, 14, 6, 15, 9, 14, true, true),
-  SMR_5_1("SMR", "5.1", 64, 11, 27, 6, 14, 6, 15, 9, 14, true, true),
-  SMR_5_2("SMR", "5.2", 67, 11, 27, 6, 14, 6, 15, 10, 15, true, true),
-  SMR_5_5("SMR", "5.5", 71, 11, 27, 6, 14, 6, 15, 10, 15, true, true);
+  DSMR_4_2_2("DSMR", "4.2.2", 59, 11, 27, 6, 14, 6, 13, 5, 6, true, false),
+  SMR_4_3("SMR", "4.3", 60, 11, 27, 6, 14, 6, 13, 6, 7, true, false),
+  SMR_5_0_0("SMR", "5.0.0", 63, 11, 27, 6, 14, 6, 15, 9, 14, true, true),
+  SMR_5_1("SMR", "5.1", 65, 11, 27, 6, 14, 6, 15, 9, 14, true, true),
+  SMR_5_2("SMR", "5.2", 68, 11, 27, 6, 14, 6, 15, 10, 15, true, true),
+  SMR_5_5("SMR", "5.5", 72, 11, 27, 6, 14, 6, 15, 10, 15, true, true);
 
```

Comment:
gaat die test niet fout als de vorige pr gemerged is?

Reply:
Ja, ik heb het aangepast